boolean-search/index.html

```diff
@@ -28,6 +28,7 @@
     <script src='../../chaise/common/config.js'></script>
     <script src='../../ermrestjs/ermrest.js'></script>
     <link rel="stylesheet" type="text/css" href="styles/booleansearch.css">
+    <script src="booleansearch-config-sample.js"></script>
```

Comment:
I know that you said this didn't break anything, but this doesn't look correct. We shouldn't use the sample file and sample file is just an example that can be used to create the actual config files.